pages/common/flake8.md

```diff
@@ -0,0 +1,20 @@
+# flake8
+
+> Tool to check the style and quality of Python code.
+> More information: <https://flake8.pycqa.org/>.
+
+- Lint a file or directory:
+
+`flake8 {{path/to/file_or_directory}}`
+
+- Show source code for each error:
```

Comment:
Does it only print the line or also the lines around it?

Reply:
It only prints the line.

Reply:
How about
```suggestion
- Show the line in which each error occured:
```
?

Reply:
Good by me. Will commit.